examples/custom-server-hapi/server.js

```diff
@@ -1,6 +1,6 @@
 const next = require('next')
 const Hapi = require('@hapi/hapi')
-const { pathWrapper, nextHandlerWrapper } = require('./next-wrapper')
+const { /* pathWrapper, */ nextHandlerWrapper } = require('./next-wrapper')
```

Comment:
```suggestion
const { nextHandlerWrapper } = require('./next-wrapper')
```

Should this be removed?

Reply:
No, I left it intentionally.